doc/build-rpi.txt

```diff
@@ -0,0 +1,29 @@
+The following dependencies and steps are needed for Raspbian Stretch released in November 2017.
+
+Prequisites may change with newer versions.
+
+This has been tested with Raspberry Pi Zero W.
+
+Note that it may be tough to get everything + the blockchain installed on a 4GB flash card.  It's recommended you use an 8GB or 16GB flash card.
+
+
+#expand the default swapfile size to 1GB since 100MB is not enough
+$ sudo nano /etc/dphys-swapfile
+#change CONF_SWAPSIZE=100 to CONF_SWAPSIZE=1024
+$ sudo /etc/init.d/dphys-swapfile stop
+$ sudo /etc/init.d/dphys-swapfile start
+
+sudo apt-get install qttools5-dev-tools
+sudo apt-get install qt5-default
+sudo apt-get install libboost-dev
+sudo apt-get install libboost-all-dev
+sudo apt-get install libssl1.0-dev
+sudo apt-get install libdb5.3++-dev
+sudo apt-get install libminiupnpc-dev
```

Comment:
Might be more practical to put all of them on the same line. Also, you don't really need `libboost-all-dev`, there are only 4 `libboost` dependencies necessary. I think you're also missing `libgmp-dev`.
Could you try something like this:
```bash
apt install qttools5-dev-tools qt5-default libboost-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev libboost-thread-dev libssl1.0-dev libdb5.3++-dev libminiupnpc-dev
```

Reply:
Yes true.  I'll do this in a future update.